Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3-alpine3.14 as base
+FROM python:3-slim as base
```

Comment:
What is the resulting change in image size here?

Reply:
`3-alpine3.14` is 45.2 MB; `3-slim` is 122 MB -- the final image is 260 MB, which is noticeably larger than the 88.1 MB of the current alpine-based build from `main`.

On the other hand, it's not quite apples-to-apples because the current code (in this branch) can't be built on any existing version of alpine at all, because compilation requires newer versions of some libraries than anything shipped by alpine. We could potentially fudge this by backleveling versions (to less backleveled than they were) in hopes of finding some happy medium, but it just didn't seem worth it.

That was before I read https://pythonspeed.com/articles/base-image-python-docker-images/ yesterday. :)